CONTRIBUTING.md

```diff
@@ -27,3 +27,14 @@ and final decision is with the Bitcoin Core project lead.
 
 Translations should be made from the English source. The English source has a `version:` field in the Front Matter which should be updated if on the rare
 occasion the English text is updated. Translated text should remove `translated: false` from the Front Matter.
+
+## License
+
+The code and documentation of this website are licensed under [MIT license][MIT] and all contributors agree to license their
+contributions under the same license.
+
+The content of this website is licensed under the Creative Commons Attribution-ShareAlike 4.0 International License: [CC-By-SA]
+and all contributors agree to license their content under the same license.
```

Comment:
I don't think we should commit to a uniform content license. We may very well wish to repost or quote others, and should be free to do so without negotiating licensing of it.